CONT-990_Adding PR template (#448)
puppetlabs/puppetlabs-accounts: .github/pull_request_template.md

## Summary
Provide a detailed description of all the changes present in this pull request.

## Additional Context

> Add any additional context about the problem here. (Root cause/steps to reproduce (if applicable), thought process on implementation)

Reviewer: ```suggestion
Add any additional context about the problem here. 
- [ ] Root cause and the steps to reproduce. (If applicable)
- [ ] Thought process on the implementation.

```

# Note : we are trying to define a template for PR, any suggestions are welcome, get in touch with CAT team...

## Summary

> Provide a detailed description of the changes in this pull request.

Reviewer: ```suggestion
Provide a detailed description of all the changes present in this pull request.
```

Author: changed

## Additional Context

> Add any other context about the problem here.

Reviewer: ```suggestion 
Add any additional context about the problem here. (Root cause/steps to reproduce (if applicable), thought process on implementation)
```

Author: changed

## Related Issues (if any)
Mention any related issues or pull requests.

## Checklist

> - [ ] Rspec.

Reviewer: ```suggestion
- [ ] 🟢 Spec tests.
```

Author: done

> - [ ] Acceptance tests.

Reviewer: ```suggestion
- [ ] 🟢 Acceptance tests.
```

Author: done

> - [ ] Manually tested.

Reviewer: ```suggestion
- [ ] Manually verified. (For example `puppet apply`)
```

Author: done
